README.md

```diff
@@ -17,10 +19,6 @@ Install with npm
 npm install tsutil --save
 ```
 
-## Documentation
```

Comment:
We should update the installation instructions too. It's missing the `npm run typings` step.